app/components/baseComponents/components/inputs/select/select.module.scss

```
@@ -0,0 +1,25 @@
+@use '../../../styles/colors' as colors;
```

nit: feels like we should have an extra newline

```suggestion
@use '../../../styles/colors' as colors;

```